js/plugins/vertexai/src/model_garden.ts

```diff
@@ -29,6 +29,23 @@ export const ModelGardenModelConfigSchema = OpenAIConfigSchema.extend({
   location: z.string().optional(),
 });
 
+export const llama32 = modelRef({
+  name: 'vertexai/llama-3.2',
+  info: {
+    label: 'Llama 3.2',
+    supports: {
+      multiturn: true,
+      tools: true,
+      media: true,
+      systemRole: true,
+      output: ['text', 'json'],
+    },
+    versions: ['meta/llama-3.2-11b-vision-instruct-maas'],
+  },
+  configSchema: ModelGardenModelConfigSchema,
+  version: 'meta/llama-3.2-11b-vision-instruct-maas',
```

Comment:
could you please double check, I believe the maas model is 90b (`meta/llama-3.2-90b-vision-instruct-maas`)

Reply:
Good catch